recipe/meta.yaml

```diff
@@ -61,7 +59,7 @@ requirements:
     - poppler                                # [build_platform != target_platform]
     - zlib                                   # [build_platform != target_platform]
     - libheif                                # [build_platform != target_platform]
-    - gettext                                # [(build_platform != target_platform) and osx]
+    - gettext                                # [build_platform != target_platform and osx]
```

Comment:
I think this is less readable

Reply:
efb19c9dc5d99f9255cdbd56be88bab29602817d